cogboard-webapp/src/components/widgets/types/BambooPlanWidget.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string, number, object } from 'prop-types';
+
+import { Typography } from '@material-ui/core';
+import { FullWidthButtonOutlined } from '../../styled';
+
+const BambooPlanWidget = props => {
+  const { url, number, lifeCycleState } = props;
+  const updated = new Date().toLocaleString();
+
+  return (
+    <>
+      <Typography
+        variant="caption">
+        <p>Updated: {updated}</p>
+        <p>State: {lifeCycleState}</p>
+      </Typography>
+      <FullWidthButtonOutlined href={url}>
+        #{number}
+      </FullWidthButtonOutlined>
+    </>
+  );
+};
+
+BambooPlanWidget.propTypes = {
+  url: object.isRequired,
```

Comment:
If prop is required, so it should always come and we don't need default in this case.

Reply:
my bad this is a string